modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPkgTestSetup.java

```diff
@@ -67,7 +67,7 @@ protected void setUpData() throws Exception {
         String sql = "CREATE TABLE ft1 (id INTEGER PRIMARY KEY, geometry BLOB)";
         run(sql);
 
-        sql = "ALTER TABLE ft1 add intProperty INTEGER";
+        sql = "ALTER TABLE ft1 add intProperty MEDIUMINT";
```

Comment:
Not clear to me why this one was changed, I mean I understand that a `` MEDIUMINT`` is an ``int``, bu did the test just keep working whit this change?

Reply:
MEDIUMINT is a 32 bits integer, INTEGER is a 64bit integer. The map to different types in Java now (the previous code was not aware of the various types of sqlite integers, from 8 to 64 bits)